src/test/java/com/indeed/jiraactions/api/customfields/TestCustomFieldValue.java

```diff
@@ -199,22 +199,22 @@ public void testSeparateWithChildFromChangelog() throws IOException {
         final String value = "Parent values: Misconfiguration(20661)Level 1 values: App Config(20669)";
         final CustomFieldValue field = apiParser.customFieldValueFromChangelog(sysadCategories, "", value);
 
-        assertEquals(field, "Misconfiguration\tApp Config");
+        assertEquals(field, "Misconfiguration\tApp Config\t");
```

Comment:
Why is this adding a tab here? It seems like this is adding an extra column here for no reason?